delegation-toolkit/get-started/delegation-quickstart.md

```diff
@@ -48,9 +48,9 @@ const bundlerClient = createBundlerClient({
 ### 3. Create a delegator account
 
 Create an account to represent Alice, the delegator who will create a delegation.
-The delegator must be a [smart account](../how-to/create-smart-account/index.md).
+The delegator must be a [smart account](../concepts/smart-accounts.md).
```

Comment:
Maybe we should say delegator must be a MetaMask smart account, as only MM account supports delegation? Or do you think it's redundant?